doc/dynamo0p3.rst

```diff
@@ -230,6 +241,44 @@ Rules
     does not require values beyond the level-1 halo. If it does then
     PSyclone will abort.
 
+Rules for Kernels that work with CMA Operators
```

Comment:
If I understand correctly, we no longer have a single set of rules for kernels, we now have 2 main types of kernels
1) traditional kernels
2) CMA kernels
where CMA kernels can be split into 3 types.
The rules appear to be different for each case.
Therefore we should have a general rule section (which we do but I don't think is general as it includes rules for traditional kernels that do not apply to certain CMA kernels.
We should also have a standard kernel section with rules specific to these types of kernels.